test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -72,9 +79,9 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              selector: ref(Fibonacci Waterfall Selector)
+              selector: Fibonacci Waterfall Selector
               traces:
-                - ref(Fibonacci Waterfall Trace)
```

Comment:
What's your thinking behind getting rid of refs? I find them to be pretty useful to really quickly see when visivo is referring to another object in a configuration block.

Reply:
My thought is that it isn't needed.  We never have a situation where we can have a string or an object that isn't a ref.